packages/tokamak/sdk/tasks/deposit-withdraw-eth.ts

```diff
@@ -0,0 +1,264 @@
+  {
+    inputs: [
+      { internalType: 'address', name: '_spender', type: 'address' },
+      { internalType: 'uint256', name: '_value', type: 'uint256' },
+    ],
+    name: 'approve',
+    outputs: [{ internalType: 'bool', name: '', type: 'bool' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [{ name: '_owner', type: 'address' }],
+    name: 'balanceOf',
+    outputs: [{ name: 'balance', type: 'uint256' }],
+    type: 'function',
+  },
+  {
+    inputs: [{ internalType: 'uint256', name: 'amount', type: 'uint256' }],
+    name: 'faucet',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+]
+const zeroAddr = '0x'.padEnd(42, '0')
+
+let addressManager = process.env.ADDRESS_MANAGER || ''
+let l1CrossDomainMessenger = process.env.L1_CROSS_DOMAIN_MESSENGER || ''
+let l1StandardBridge = process.env.L1_STANDARD_BRIDGE || ''
+let optimismPortal = process.env.OPTIMISM_PORTAL || ''
+let l2OutputOracle = process.env.L1_OUTPUT_ORACLE || ''
+
+const updateAddresses = async (hre: HardhatRuntimeEnvironment) => {
+  if (addressManager === '') {
+    const Deployment__AddressManager = await hre.deployments.get(
+      'AddressManager'
+    )
+    addressManager = Deployment__AddressManager.address
+  }
+
+  if (l1CrossDomainMessenger === '') {
+    const Deployment__L1CrossDomainMessenger = await hre.deployments.get(
+      'L1CrossDomainMessengerProxy'
+    )
+    l1CrossDomainMessenger = Deployment__L1CrossDomainMessenger.address
+  }
+
+  if (l1StandardBridge === '') {
+    const Deployment__L1StandardBridge = await hre.deployments.get(
+      'L1StandardBridgeProxy'
+    )
+    l1StandardBridge = Deployment__L1StandardBridge.address
+  }
+
+  if (optimismPortal === '') {
+    const Deployment__OptimismPortal = await hre.deployments.get(
+      'OptimismPortalProxy'
+    )
+    optimismPortal = Deployment__OptimismPortal.address
+  }
+
+  if (l2OutputOracle === '') {
+    const Deployment__L2OutputOracle = await hre.deployments.get(
+      'L2OutputOracleProxy'
+    )
+    l2OutputOracle = Deployment__L2OutputOracle.address
+  }
+}
+
+const depositETH = async (amount: NumberLike) => {
+  console.log('Deposit TON:', amount)
+  console.log('l1 address:', l1Wallet.address)
+  console.log('l2 address:', l2Wallet.address)
+
+  const ethContract = new ethers.Contract(predeploys.ETH, erc20ABI, l2Wallet)
+
+  const l1Contracts = {
+    StateCommitmentChain: zeroAddr,
+    CanonicalTransactionChain: zeroAddr,
+    BondManager: zeroAddr,
+    AddressManager: addressManager,
+    L1CrossDomainMessenger: l1CrossDomainMessenger,
+    L1StandardBridge: l1StandardBridge,
+    OptimismPortal: optimismPortal,
+    L2OutputOracle: l2OutputOracle,
+  }
+  console.log('l1 contracts:', l1Contracts)
+
+  const bridges = {
+    TON: {
+      l1Bridge: l1Contracts.L1StandardBridge,
+      l2Bridge: predeploys.L2StandardBridge,
+      Adapter: TONBridgeAdapter,
+    },
+  }
```

Comment:
```suggestion
  const bridges = {
    ETH: {
      l1Bridge: l1Contracts.L1StandardBridge,
      l2Bridge: predeploys.L2StandardBridge,
      Adapter: ETHBridgeAdapter,
    },
  }
```

Reply:
I think ETHBridgeAdapter is for legacy usage,